### PR TITLE
update MM-EE to use 5.28.0

### PR DESCRIPTION
**Pull request by erezo9**

### CODE REVIEW

**Review by cpanato — CHANGES_REQUESTED** [review]
please one two PRS one for TE and another for EE

File: charts/mattermost-enterprise-edition/values.yaml [hunk 1]
```diff
@@ -97,7 +97,7 @@ mattermostApp:
   replicaCount: 2
   image:
     repository: mattermost/mattermost-enterprise-edition
-    tag: 5.27.0
+    tag: 5.28.0
```
Comment (cpanato) [inline charts/mattermost-enterprise-edition/values.yaml:100]: please bump, was released yesterday
```suggestion
    tag: 5.28.1
```

**Review by cpanato — CHANGES_REQUESTED** [review]

File: charts/mattermost-enterprise-edition/values.yaml (hunk 1 above)
Comment (erezo9) [reply charts/mattermost-enterprise-edition/values.yaml]: Done also changed in the Chart.yaml as well

**Review by cpanato — APPROVED** [review]
thank you @erezo9 are you willing to open a PR in for the TE as well?

**Pull request merged**